### ISSUE TITLE
AngularFireAnalytics logEvent does not output to console when debug mode is enabled

### ISSUE THREAD

**Issue by gmaughan** [angular/angularfire#2311]
When logging a custom event with `logEvent()` nothing is printed to the console to indicate that the event is sent. This would help with debugging. My event is sent to Analytics. I just have to login to see it.

I've enabled debug mode and standard `screen_view` events do appear in the console, just not 'custom events'.

I'm using the following in my `ionViewWillEnter()` of my dialog page to log an event for the user opening a dialog:

```javascript
  await this.analytics.setCurrentScreen('select-icon');
  await this.analytics.updateConfig({
      page_path: 'select-icon',
      screen_class: 'app-select-icon'
    });
  await this.analytics.logEvent('screen_view');`
```

The dialog is created using Ionic's `ModalController` and therefore doesn't go through the router. I'm assuming it is the right way to do it as I couldn't find any guidance on it. In particular, should I be calling these methods using `zone.runOutsideAngular()`.

The code looks like it only logs from a `tap` in `ScreenTrackingService`. Perhaps this `tap` statement could be moved up to a `logEvent` proxy method to log all events if debug is on.

Angular: 8.2.14
Ionic: 4.11.10
AngularFire: 5.4.0

**Comment by jamesdaniels** [comment]
Excellent feedback, thank you! I'll address in v6

**Comment by jamesdaniels** [comment]
Should be addressed in `6.1.0-rc.3`
